### ISSUE TITLE
BuildHostSecretsTask throws `error : Specified cast is not valid.`

### ISSUE THREAD

**Issue by dansiegel** [dansiegel/Mobile.BuildTools#6]
This has been reported several times. Unfortunately it the error isn't giving much in the way of information about the what was being cast.

```
Target "BuildHostSecrets" in file "/Users/dansiegel/.nuget/packages/mobile.buildtools/1.0.5/build/Mobile.BuildTools.targets":
  Using "BuildHostSecretsTask" task from assembly "/Users/dansiegel/.nuget/packages/mobile.buildtools/1.0.5/build/Mobile.BuildTools.dll".
  Task "BuildHostSecretsTask"
    Output Path: /Users/dansiegel/Projects/Demo1/Demo1/secrets.json
    /Users/dansiegel/.nuget/packages/mobile.buildtools/1.0.5/build/Mobile.BuildTools.targets(41,5): error : Specified cast is not valid.
  Done executing task "BuildHostSecretsTask" -- FAILED.
```

**Comment by chrisgate** [comment]
Thanks

**Comment by dansiegel** [comment]
This has been fixed and should be available shortly on NuGet in v1.0.6
